sandboxnu/graduatenu
# Setup running new backend

# Description

This is the first step towards connecting the frontend to the new backend. 

These changes add a script(`yarn start:v2`) in the root directory that runs both the frontend and backend together at http://localhost:3002 by proxying any request with the `/api` path to the new server and the rest to the frontend.

Closes: https://trello.com/c/P3rTw5fo/375-setup-frontend-and-backend-to-run-together-on-dev

## Type of change

Please delete options that are not relevant.


# How Has This Been Tested?

-  Run `yarn db:up` to start the dev db and then `yarn start:v2` to run the application with the new backend. Visit http://localhost:3002 to view the new application. It won't work yet since the backend isn't fully connected to the frontend yet but you should be able to see the app and visiting `/api` should give you a 404 from the new api. Once you're done run `yarn db:down` to stop the dev db.

- Run `yarn start` to run the old server with the app. Everything should function like before.


# Checklist:
